startup.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 bundle install
-bundle exec rails s -p 3205
+bundle exec foreman start
```

Comment:
This won't run it with the correct port by default. We could prefix it with `PORT=3205` to solve that though.